### PR TITLE
Add TeamSpeak as Hard

### PR DESCRIPTION
**Pull request by Marilok**

### CODE REVIEW

**Comment by SvicidalBug** [comment]
Entries need to be in alphabetical order. Could you move `TeamSpeak` to come just before `TeamViewer`?

File: _data/sites.json [hunk 1]
```diff
@@ -11184,7 +11184,18 @@
             "teamgantt.com"
         ]
     },
-
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:11188]:
```suggestion

    {
```

File: _data/sites.json [hunk 2]
```diff
@@ -11184,7 +11184,18 @@
...
+        "name": "TeamSpeak",
+        "url": "https://support.teamspeak.com/hc/en-us/articles/4408933645201-I-can-t-find-the-option-to-delete-myTeamSpeak-account-How-do-I-do-this-",
+        "difficulty": "hard",
+        "notes": "You need to contact the support using their contact form.",
+        "domains": [
+            "teamspeak.com",
+            "myteamspeak.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:11198]:
```suggestion
    },

```

File: _data/sites.json [hunk 3]
```diff
@@ -11199,7 +11210,7 @@
             "teamviewer.com"
         ]
     },
-
+
```
Comment (tupaschoal) [inline _data/sites.json:11213]:
```suggestion

```

File: _data/sites.json [hunk 4]
```diff
@@ -11199,7 +11210,6 @@
             "teamviewer.com"
         ]
     },
```
Comment (tupaschoal) [inline _data/sites.json:11212]:
```suggestion
    },

```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**